plugins/woocommerce/tests/e2e-pw/tests/smoke-tests/update-woocommerce.spec.js

```diff
@@ -7,115 +8,123 @@ const {
 	deleteZip,
 } = require( '../../utils/plugin-utils' );
 
-const pluginZipPath = path.resolve( __dirname, '../../tmp/woocommerce.zip' );
+let pluginZipPath;
 
-test.describe( 'WooCommerce plugin can be uploaded and activated', () => {
-	// Skip test if UPDATE_WC is falsy.
-	test.skip(
-		! Boolean( UPDATE_WC ),
-		`Skipping this test because UPDATE_WC is falsy: ${ UPDATE_WC }`
-	);
+test.describe.serial(
```

Comment:
I had to use `serial` here so that the 2nd test (`can run the database update`) will be skipped if the first test (`can upload and activate the WooCommerce plugin`) fails. 

Without it, there will be cases where the db update (the 2nd test) will be executed even though the uploading of the WC nightly build failed.